pilot/pkg/networking/core/v1alpha3/gateway.go

```diff
@@ -156,7 +156,7 @@ func (configgen *ConfigGeneratorImpl) buildGatewayListeners(builder *ListenerBui
 			}
 
 			for cnum := range mutable.FilterChains {
-				if util.IsIstioVersionGE117(builder.node.IstioVersion) {
+				if util.IsIstioVersionGE117(builder.node.IstioVersion) && features.NetworkAuthn {
```

Comment:
Do we need to update https://github.com/istio/istio/blob/534e4fa90e4d6b316061e57a44f254d0376b978b/pilot/pkg/networking/plugin/authz/authorization.go#L52? Or is this also disabling authn entirely?

Reply:
I think this is for ingress only (CloudESF), no sidecars.

Reply:
Ack, may be nice to clarify in the env var description that this will mean any authz policies using identity will never match? That is the impact, right?

Reply:
+1 on adding this to the flag comment. Maybe a link to a doc or wiki page - "Using Istio with upstream Envoy". I assume WASM or EnvoyFilters or a subset of Authz can still be used ? This needs to be documented.